routes/index.js

```diff
@@ -269,7 +269,7 @@ router.post('/api/move-envelope', (req, res) => {
   const sql = `UPDATE ENVELOPES 
              SET seat_number = ${db.escape(req.body.nextSeat)}, 
                 envelope_state = 0,
-                prev_completed = IF(${db.escape(req.body.nextSeat)} = 3, true, false)
+                prev_completed = IF(${db.escape(req.body.nextSeat)} = 3, true, IF(prev_completed = true, true, false))
```

Comment:
maybe this should be a larger discussion, but are we concerned about properly parameterizing SQL queries here to avoid possible SQL injection issues?

Reply:
We discussed this with Scribbles and we determined that what we had right now was sufficient

Reply:
The other SQL we were looking at are ok but this one should be parameterizing data to make sure it is escaped. Let me know if you want some help with this.

Reply:
SQL Injection prevention will be worked on in ENG-1994